.github/workflows/QA.yml

```diff
@@ -11,6 +11,12 @@ on:
       - dev
   pull_request:
   workflow_dispatch:
+    inputs:
+      debug_enabled:
+        type: boolean
```

Comment:
should the leave this in? it is actually really useful for debugging - but if we leave it in you do have to disable that forked repos can trigger CI without review from the core team. Otherwise it is a security issue